schulcloud/FilesViewController.swift

```diff
@@ -119,6 +119,10 @@ class FilesViewController: UITableViewController, NSFetchedResultsControllerDele
 
     override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
         guard let currentUser = Globals.currentUser else { return false }
+        guard let file = self.fetchedResultsController.sections?[indexPath.section].objects?[indexPath.row] as? File,
+              file.id != FileHelper.rootDirectoryID, file.parentDirectory?.id != FileHelper.rootDirectoryID, file.parentDirectory?.id != FileHelper.userDirectoyID
```

Comment:
Are you sure about this? `file.parentDirectory?.id != FileHelper.userDirectoyID`
Doesn't this prevent the user from editing the files in this personal folder?

Reply:
Omg, good catch, i meant to check the course.